prosodicHierarchy.js

```diff
@@ -29,6 +58,17 @@ var sCat = ["cp", "xp", "x0"];
 //defines the prosodic hierarchy
 var pCat = ["u", "i", "phi", "w", "Ft", "syll"];
 
+//defines Major Minor Phrase prosodic hierarchy
+var MaPMiPpCat = ["i", "MaP", "MiP", "w"];
+
+function setPCat(newPCat){
+	pCat = newPCat;
```

Comment:
We need to add the private functions pCat.isHigher and pCat.isLower to this too, I think.